test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -268,6 +268,7 @@ static inline void multi_access_publisher(bool intra_process)
     .use_intra_process_comms(intra_process);
 
   auto node = rclcpp::Node::make_shared(node_topic_name, options);
+  printf("Node initialized");
```

Comment:
Mind removing this print statement? It seems unrelated.

Reply:
Done